ydb/core/kqp/executer_actor/kqp_scheme_executer.cpp

```diff
@@ -120,6 +122,15 @@ class TKqpSchemeExecuter : public TActorBootstrapped<TKqpSchemeExecuter> {
         Become(&TKqpSchemeExecuter::ExecuteState);
     }
 
+    TString GetDatabaseName() const {
```

Comment:
There is already common (for kqp) method for that -- `NKikimr::NKqp::NSchemeHelpers::SetDatabaseForLoginOperation`, see [ydb/core/kqp/gateway/utils/scheme_helpers.cpp#L59](https://github.com/ydb-platform/ydb/blob/main/ydb/core/kqp/gateway/utils/scheme_helpers.cpp#L59).

Please consider using it instead.